example/lib/main.dart

```diff
@@ -45,7 +45,8 @@ class _HomeState extends State<Home> with SingleTickerProviderStateMixin {
   /// Initialization of controllers.
   late TabController _tabController;
   final userAttr = {"id": Platform.isIOS ? "foo" : "foo_bar"};
-  late final GrowthBookSDK gb;
+  GrowthBookSDK? gb;
```

Comment:
This appears to be a change that is treating a symptom. Is there a reason why this is not initialized? Is this a result of something that was changed or has it always been a problem in this example?

Reply:
It shouldn't be a late variable because we need to access it before it has time to be initialized. The fact that `setState()` is being called is proof we access it before initialization has finished since we call `setState()` after it is done.